roles/network_plugin/calico/templates/calico-cr.yml.j2

```diff
@@ -13,6 +13,16 @@ rules:
       - configmaps
     verbs:
       - get
+{% if calico_version is version("v3.20.0", ">=") %}
+  # EndpointSlices are used for Service-based network policy rule
+  # enforcement.
+  - apiGroups: ["discovery.k8s.io"]
+    resources:
+      - endpointslices
+    verbs:
+      - watch
+      - list
```

Comment:
No need for `get` ?

Reply:
According to upstream manifests no, only `watch` and `list` are needed. See https://github.com/projectcalico/calico/blob/v3.20.0/_includes/charts/calico/templates/calico-node-rbac.yaml#L16-L23